RallySoftware/agile-central-ckeditor
# perf(lang): S126655 - Support for English (all dialects), Brazilian Portuguese, Simplified Chinese, and German only

#### What does this PR do?

Removes support for a bunch of languages CA Agile Central currently doesn't support.

For the CKEditor Build
Before optimization:
```
Number of files: 2969
Total size.....: 5731674 bytes
Time taken.....: 71.102seconds
```
After optimization:
```
Number of files: 862
Total size.....: 4269215 bytes
Time taken.....: 27.86seconds
```

#### Any background context you want to provide?

Our deploy process is bogged down by shipping copies of language packs we don't use. Multiply that by the number of versions of the front-end we have and the result is slower deploys.

#### Where should the reviewer start?

ckeditor-build-config.js

#### How should this be manually tested?

Change the language on your computer, close and open browsers, and then open sample.html in the supported language on your machine/browser. Those languages should still work.

If you select an unsupported language, Japanese for example, the CKEditor should default the language to English.
